lib/tasks/files/sudo.debian

```diff
@@ -0,0 +1,3 @@
+Defaults:www-data !requiretty, !authenticate
+Defaults:www-data env_keep += "NGINX_STAGE_* OOD_*"
+www-data ALL=(ALL) NOPASSWD: /opt/ood/nginx_stage/sbin/nginx_stage
```

Comment:
We also add these to RPM deployed sudo file:
```
Cmnd_Alias KUBECTL = /usr/local/bin/kubectl, /usr/bin/kubectl, /bin/kubectl
Defaults!KUBECTL !syslog
```

Reply:
thx. I'd been looking at what was ~2.10.